src/Aspire.Hosting.Milvus/MilvusBuilderExtensions.cs

```diff
@@ -56,27 +54,6 @@ public static IResourceBuilder<MilvusServerResource> AddMilvus(this IDistributed
             ParameterResourceBuilderExtensions.CreateDefaultPasswordParameter(builder, $"{name}-key");
 
         var milvus = new MilvusServerResource(name, apiKeyParameter);
-
-        MilvusClient? milvusClient = null;
-
-        builder.Eventing.Subscribe<ConnectionStringAvailableEvent>(milvus, async (@event, ct) =>
-        {
-            var connectionString = await milvus.ConnectionStringExpression.GetValueAsync(ct).ConfigureAwait(false)
-            ?? throw new DistributedApplicationException($"ConnectionStringAvailableEvent was published for the '{milvus.Name}' resource but the connection string was null.");
-            milvusClient = CreateMilvusClient(@event.Services, connectionString);
-        });
-
-        var healthCheckKey = $"{name}_check";
-        // TODO: Use health check from AspNetCore.Diagnostics.HealthChecks once it's implemented via this issue:
-        // https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/issues/2214
-        builder.Services.AddHealthChecks()
```

Comment:
What other Hosting integration packages don't have health checks? Will this one be unique?

Reply:
No it will not be unique. Seq, Qdrant